defaults.yaml

```diff
@@ -12,7 +12,10 @@ conf:
   minion_1_name: k8s-minion-1
   minion_2_name: k8s-minion-2
   minion_3_name: k8s-minion-3
-  kubernetes_nodes_base_box_id: "bento/centos-7.4"
+  vagrant_provider: libvirt
```

Comment:
This should not be the default. Revert to virtualbox and eventually configure libvirt in your env.yaml.